src/Compilers/CSharp/Portable/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -154,6 +154,13 @@ internal static RefKind GetRefKind(this TypeSyntax syntax)
             return refKind;
         }
 
+        internal static TypeSyntax SkipRef(this TypeSyntax syntax)
```

Comment:
Is this overload necessary? It looks like there is only one caller that was previously calling the other overload.